src/server/controllers/user.js

```diff
@@ -1,39 +1,62 @@
-const bcrypt = require('bcrypt');
-const jwt = require('jsonwebtoken');
-const { PrismaClient } = require('@prisma/client');
-const prisma = new PrismaClient();
+const bcrypt = require('bcrypt')
+const jwt = require('jsonwebtoken')
+const { PrismaClient } = require('@prisma/client')
+const prisma = new PrismaClient()
 
-const jwtSecret = 'mysecret';
+const jwtSecret = 'mysecret'
 
 const register = async (req, res) => {
-    const { username, password } = req.body;
-
-    const createdUser = null;
-
-    res.json({ data: createdUser });
-};
+  const { username, password } = req.body
+
+  const saltrounds = 10
+
+  bcrypt.hash(password, saltrounds, async (err, hashedPwd) => {
+    try {
+      const createdUser = await prisma.user.create({
+        data: {
+          username: username,
+          password: hashedPwd
+        }
+      })
+      delete createdUser.password
```

Comment:
I've learned that this `delete` keyword belongs to the Object constructor in JavaScript.
Nice to see it being used here to delete a key of `createdUser` obj